src/routes/safe/components/Apps/hooks/useIframeMessageHandler.ts

```diff
@@ -0,0 +1,52 @@
+import { useEffect } from 'react'
+
+const useIframeMessageHandler = (): void => {
+  useEffect(() => {
```

Comment:
why is this code commented? is it still WIP?

Reply:
PR description:

> TODO:
> 
> Add useIframeMessageHandler hook for communication login between app <> safe app. This PR is already pretty big so I decided to work on it in a separate PR

Reply:
sorry, missed that!